sites/topomath.asu.edu/modules/system_descriptions/static/sysDescUpdates.php

```diff
@@ -0,0 +1,107 @@
+<?php
+//define('DRUPAL_ROOT', '/Applications/XAMPP/htdocs/dragoon-lms');
+$drp_root_handle=fopen("drupal_root","r");
+$drupal_root = fgets($drp_root_handle);
+fclose($drp_root_handle);
+define('DRUPAL_ROOT', trim($drupal_root));
+require_once "".DRUPAL_ROOT . "/includes/bootstrap.inc";
+drupal_bootstrap(DRUPAL_BOOTSTRAP_FULL);
+$request_type = $_REQUEST["req_type"];
+takeAction($request_type);
+
+//All take actions require a working db connection
+function takeAction($case=" "){
+
+	switch ($case) {
+		case "newDesc":
+			//add the new description to data base from the input parameters
+			$name = $_REQUEST['n'];
+			$user = $_REQUEST['u'];
+			$section = $_REQUEST['s'];
+			$folder = $_REQUEST['f'];
+			$prob = $_REQUEST['p'];
```

Comment:
the common fields in this switch case can be made a variable before the switch case. That way if you want to change the parameter for request you wont have to do it at every place.